src/Vulkan/VMASharp_API_For_Review.md

```diff
@@ -0,0 +1,307 @@
+```cs
+namespace VMASharp.Metadata
+{
+  public interface IBlockMetadata
+  {
+    long Size { get; }
+    int AllocationCount { get; }
+    long SumFreeSize { get; }
+    long UnusedRangeSizeMax { get; }
+    bool IsEmpty { get; }
+    void Validate();
```

Comment:
i guess since it is void this throws if it is invalid ?

Reply:
Yes, that is the design choice, but its only called in Debug Builds